grammar.js

```diff
@@ -649,6 +680,36 @@ module.exports = grammar({
       $._statement,
     ),
 
+    // This is missing binary expressions, others were kept so that macro code can be parsed better and code examples
```

Comment:
would be good to say why we skipped binary expressions. what's the problem with binary expression at the top?

Reply:
If it's because of this ambiguity with pointer expressions, could have been solved in another way? by playing with dynamic priorities?

Reply:
Yeah I did try a lot w/ dynamic precendences, but that caused other issues elsewhere mainly and sometimes didn't do anything as well